/lib/cpp might not exist outside $EPREFIX
Bart Oldeman <bart.oldeman@calculquebec.ca>
--- ncl-6.6.2/config/ymake.orig	2025-05-06 12:59:24.265622748 +0000
+++ ncl-6.6.2/config/ymake	2025-05-06 12:59:58.242788802 +0000
@@ -639,7 +639,7 @@
 else if (-e /usr/bin/cpp) then
     set cpp = /usr/bin/cpp
 else
-  set cpp = /lib/cpp
+  set cpp = $EPREFIX/lib/cpp
 endif
 
 # set cpp options
